### PR TITLE
 infoschema: add mem table to show hot region

### LINKED ISSUE #9885: Support query hot read / write regions by SQL
## Feature Request

It would be great if we can directly use SQL to query hot read and hot write regions.

The respond schema may be like:

```
region id | database id | database name | table id | table name | index id (if any) | index name (if any)
```

### PR DESCRIPTION
**Pull request by winoros**
### What problem does this PR solve? 

To implement and close #9885

### What is changed and how it works?

Add a mem table to show the hot region information.
```sql
CREATE TABLE `TIDB_HOT_REGIONS` (
  `TABLE_ID` bigint(21) unsigned DEFAULT NULL,
  `INDEX_ID` bigint(21) unsigned DEFAULT NULL,
  `DB_NAME` varchar(64) DEFAULT NULL,
  `TABLE_NAME` varchar(64) DEFAULT NULL,
  `INDEX_NAME` varchar(64) DEFAULT NULL,
  `TYPE` varchar(64) DEFAULT NULL,
  `MAX_HOT_DEGREE` bigint(21) unsigned DEFAULT NULL,
  `REGION_COUNT` bigint(21) unsigned DEFAULT NULL,
  `FLOW_BYTES` bigint(21) unsigned DEFAULT NULL
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_bin
```
### Check List 

Tests 

 - Manual test on a existing cluster.
 - No code

Code changes

 - Has exported function/method change
 - Has exported variable/fields change

Related changes

 - Need to update the documentation
 - Need to be included in the release note

### CODE REVIEW

**Comment by winoros** [comment]
Why there's no unit test in package `information_schema` to test it?

The function needs get region information from region cache.
It's a little hard to mock it together with the mock tikv. Since the final output contains  `TableName` which need to first get table id then get its name by `InforSchema`.
This means we need to first create a table with a testkit. Then get its table id and find a way mock the region cache so that the cache can find the correct table id by the region id we return from the mock http server.

**Comment by winoros** [comment]
There're many changes to resolve the cycle import.

And seems that there's still cycle import in tests...

File: util/pdapi/const.go [hunk 1]
```diff
@@ -0,0 +1,7 @@
+package pdapi
```
Comment (lamxTyler) [inline util/pdapi/const.go]: Miss license.

File: store/helper/helper.go [hunk 2]
```diff
@@ -133,3 +145,223 @@ func (h *Helper) FetchHotRegion(rw string) (map[uint64]RegionMetric, error) {
...
+					return f
+				}
+			}
+		}
+	}
+	return nil
+}
+
+// NewRegionFrameRange init a NewRegionFrameRange with region info.
+func NewRegionFrameRange(region *tikv.KeyLocation) (idxRange *RegionFrameRange, err error) {
```
Comment (lamxTyler) [inline store/helper/helper.go]: Should we remove the following functions from `http_handler.go`?

**Pull request merged**